### PR TITLE
Rubocop

### LINKED ISSUE #35: Add Rubocop
I hate it when I have to look back at ugly code that previous-me wrote. Rubocop will at least make it  consistent.

### PR DESCRIPTION
**Pull request by stephengroat**
closes #35

### CODE REVIEW

**Review by titusfortner — APPROVED** [review]
I don't love the disabling of the metrics, but the code needs a refactor, and adjusting those can be part of it.

Let's merge this. Thanks for the implementation!

@kapoorlakshya or @stephengroat would one of you like to add a Travis job for this?

**Comment by stephengroat** [comment]
Since the rubocop config just yaml, you can add `# TODO` comments to the file and turn on something like https://GitHub.com/JasonEtco/todo

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
+require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
+
+RuboCop::RakeTask.new
 
 RSpec::Core::RakeTask.new(:spec)
 
-task :default => :spec
+task default: %w[spec rubocop]
```
Comment (stephengroat) [inline Rakefile]: @titusfortner this change to the default rake task runs rubocop on each travis run

**Comment by kapoorlakshya** [comment]
@stephengroat I just merged #50 and it caused a conflict here. Can you please resolve it? I'll merge this once all checks pass again. Thank you!

**Pull request closed without merging**